portals/api-control-plane/bff/app/app_test.go

```
@@ -0,0 +1,77 @@
+			}),
+		},
+	})
+	if err != nil {
+		t.Fatalf("app.New: %v", err)
+	}
+	defer srv.Close()
+
+	ts := httptest.NewServer(srv.Handler())
+	defer ts.Close()
+
+	res, err := http.Get(ts.URL + "/api/environments")
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)